app/src/main/java/com/tomtom/sdk/examples/usecase/BasicNavigationActivity.kt

```diff
@@ -283,7 +278,9 @@ class BasicNavigationActivity : AppCompatActivity() {
             routeOffset = route.routePoints.map { it.routeOffset }
         )
         tomTomMap.addRoute(routeOptions)
-        tomTomMap.zoomToRoutes(ZOOM_TO_ROUTE_PADDING)
+        if (withZoom) {
+            tomTomMap.zoomToRoutes(ZOOM_TO_ROUTE_PADDING)
```

Comment:
I would just call `zoomToRoute` in places where it is needed to keep it simple stupid for now. Because now there is exactly one place where we need to call zoom after drawing a line, so with this flag here we won't reduce code duplication.

Reply:
@AdrianGlasnapp-TomTom the desired change has been made.